administration/src/bp-modules/applications/ApplicationsOverview.tsx

```diff
@@ -68,7 +68,7 @@ const ApplicationsOverview = ({ applications }: { applications: Application[] })
     () =>
       sortedApplications.filter(application => {
         if (activeBarItem.status === undefined) {
-          return application
+          return Boolean(application)
```

Comment:
:wrench: Please avoid typecasts like this. It is really implicit in its behavior and usually there is a reason why we get a ts/linting error.
```suggestion
          return true
```